pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -1414,7 +1414,7 @@ ASTSqlExpression SqlExpression() :
  |
     LOOKAHEAD(2) <ROWNUM>
  |
-    LOOKAHEAD(2) Literal()
+    LOOKAHEAD(2) AdditiveExpression() // this can be a literal or a simple expression, but no conditional
```

Comment:
> this can be a literal or a simple expression, but no conditional

Maybe we should add a test asserting that using a conditional fails? I'm not sure whether we have negative parsing tests yet. Maybe it's worth it also to remove the duplication of testing logic and make it easier to add new tests. Or alternatively, use pmd/pmd-regression-tester to parse real-world PLSQL

Reply:
the comment is more a reminder for me...
I'm actually taking a short-cut here, reusing the plsql-expression in a sql-expression - the sql-expression is more restrictive, that's why we can't use "Expression()" here.

I don't think, we need an extra (negative) parsing test here. I might change this SqlExpression in the future.
Using pmd-regression-tester is definitively what we should use.